### ISSUE TITLE
Add Zoom functionality to Explore

### ISSUE THREAD

**Issue by tanvidadu** [commons-app/apps-android-commons#1770]
**Summary:** 

Add functionality to zoom images displayed after browsing through app ( Just like share Activity) . 

**Would you like to work on the issue?**
If no one takes it up, I will work on it

**Comment by nicolas-raoul** [comment]
Thanks!
Just yesterday I was thinking we should add this, because many people want to view the whole picture in full resolution without actually having to press the "Download" button and viewing it with a different app as that is cumbersome and generates a lot of useless files on the SD card.

**Comment by ujjwalagrawal17** [comment]
Yup, I was also thinking of this feature. I also created 1 issue similar to this #1147  

The Wikipedia app displays the image in a very clean way. It would be nice if we can have it similar to them.

**Screenshot:** 
![screenshot_20180731-211025](https://user-images.githubusercontent.com/19607555/43496458-16136be0-955b-11e8-9b20-5bdbcad41f6b.png)

A next step after this can be adding viewPager support in this feature similar to android gallery.

**Comment by tanvidadu** [comment]
For adding zoom functionality , should I add a zoom option in menubar or simply add pinch zoom functionality ? 
I prefer first option but I am open to second one too! I would love to hear others opinion too :)

**Comment by misaochan** [comment]
For browsing images, I'd vote for pinch zoom, but I'm OK with the menubar if others prefer that. :)

**Comment by domdomegg** [comment]
@tanvidadu Are you still working on this?

**Comment by tanvidadu** [comment]
Nope due to some prior commitments, I am unable to work on it!
@domdomegg feel free to take it up :)

**Comment by domdomegg** [comment]
No problem - but I'm not planning on taking this up so still unassigned. Thank you for the update and for removing assigned tag though :)
